force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -32,7 +32,12 @@ public with sharing class VariablesDataTypesOperators {
      * @return The sum of the two numbers, or null if either number is null.
      */
     public static Integer addition(Integer a, Integer b) {  
-        return null; // Replace null with the variable you used to store the result
+        if (a == null || b == null) {
+            return null;
+        }
+        else {
+            return a+b;
```

Comment:
```suggestion
            return a + b;
```
The extra space will help a little with readability.
Also for practice, try to use a ternary operator.